lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BaseClient } from "./BaseClient";
+import { AuthorizationCodeUrlRequest } from "../request/AuthorizationCodeUrlRequest";
+import { AuthorizationCodeRequest } from "../request/AuthorizationCodeRequest";
+import { Authority } from "../authority/Authority";
+import { RequestUtils } from "../utils/RequestUtils";
+import { RequestValidator } from "../request/RequestValidator";
+import { GrantType } from "../utils/Constants";
+import { Configuration } from "../config/Configuration";
+
+/**
+ * Oauth2.0 Authorization Code client
+ */
+export class AuthorizationCodeClient extends BaseClient {
+
+    constructor(configuration: Configuration) {
+
+        super({
```

Comment:
is there a reason we would not just pass the whole configuration object here? looks to have the same structure

Reply:
Yep you're right, should be passing configuration object instead - updated both here and DeviceCodeClient